.dotnet/src/Polyfill/System.Diagnostics.CodeAnalysis.ExperimentalAttribute.cs

```diff
@@ -0,0 +1,62 @@
+// <auto-generated/>
```

Comment:
I see the `// <auto-generated>` comment. Was this polyfill auto-generated using a tool? Would it be better if we just manually grab the code directly from here:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/ExperimentalAttribute.cs